pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -152,14 +152,14 @@ func (s *Server) initDNSCerts(hostname, customHost, namespace string) error {
 
 			// When Citadel is configured to use self-signed certs, keep a local copy so other
 			// components can load it via file (e.g. webhook config controller).
-			if err := os.MkdirAll(dnsCertDir, 0700); err != nil {
+			if err := os.MkdirAll(dnsCertDir, 0o700); err != nil {
```

Comment:
Why "0700" needs to be changed into "0o700"?

Reply:
this is just the auto formatter kicking in. I think since we upgraded to golang 1.16 its now formatting this way